views/js/e2e/runner/_setup/setupSelectors.js

```diff
@@ -32,5 +32,12 @@ export default {
     deliveriesPage: {
         rootDeliveryClass: 'li[data-uri="http://www.tao.lu/Ontologies/TAODelivery.rdf#AssembledDelivery"]',
         formContainer: '#form-container'
+    },
```

Comment:
I would separate this to navigationSelectors or testSelectors file

Reply:
yes